docs/reference/contrib/forms/customization.md

```diff
@@ -764,11 +764,11 @@ class FormPage(AbstractEmailForm):
 
 ## Custom `clean_name` generation
 
--   Each time a new `FormField` is added a `clean_name` also gets generated based on the user entered `label`.
--   `AbstractFormField` has a method `get_field_clean_name` to convert the label into a HTML valid `lower_snake_case` ASCII string using the [AnyAscii](https://pypi.org/project/anyascii/) library which can be overridden to generate a custom conversion.
+-   Each time a new `FormField` is added a `clean_name` also gets generated based on the user-entered `label`.
+-   `AbstractFormField` has a method `get_field_clean_name` to convert the label into an HTML valid `lower_snake_case` ASCII string using the [AnyAscii](https://pypi.org/project/anyascii/) library which can be overridden to generate a custom conversion.
```

Comment:
I think `a` (the original) is more correct. 

https://www.scribbr.com/commonly-confused-words/a-vs-an/

Reply:
I disagree - in formal English the H in HTML is pronounced "aitch", not "haitch" (even if that's not true for all accents...) so the "an" is needed to match that. I'd also put a hyphen in "HTML-valid" though.